_sources/Sequences/Exercises.rst

```diff
@@ -0,0 +1,46 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: seq-11-
+   :start: 1
+
+Exercises
+---------
+
+#.
+
+   .. parsonsprob:: pp5_11_1
+
+      Write a program that will print out the length of each item in the list as well as the first and last characters of the item.
+      -----
+      weather = ["sunny", "cloudy", "partially sunny", "rainy", "storming", "windy", "foggy", "snowy", "hailing"]
```

Comment:
can you brake up the long lists into multiple lines please.  This really makes the parsons problem look ugly on my browser.